GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -22,6 +22,8 @@ public class PlayInstantQuickDeployWindow : EditorWindow
         private static int _toolbarSelectedButtonIndex = 0;
```

Comment:
Nit: flip order of this and next to keep non-readonly fields together

Reply:
Resolved.